File: lv1/푸드파이트대회.py
def solution(food):
  del food[0]
  dir = {}
  for index, n in enumerate(food):
    if n//2 != 0:
      dir[index + 1] = n//2
  arr = []
  for key, val in dir.items():
    n = 0
    while n < val:
      arr.append(str(key))
      n += 1
  arr2 = list(reversed(arr))
  answer = "".join(arr) + "0" + "".join(arr2)
  return answer

File: lv1/푸드파이트대회.js
function solution(food) {
  var answer = '';
  let str1 = '';
  food.shift();
  food.forEach((e, i) => {
    for (let a = 0; a < Math.floor(e / 2); a++) {
      str1 += i + 1;
    }
  })
  answer += str1;
  answer += '0';
  answer += [...str1].reverse().join("");
  return answer;
}